### DISCUSSION TITLE
Are videos included in-line?

### DISCUSSION THREAD

**Discussion by veitch123** [ReagentX/imessage-exporter#93] [Q&A] [answered]
Apologies but i couldn't see anything about videos.

**Comment by ReagentX** [comment, marked as answer]
Yes. See the attachments section in [features.md](https://github.com/ReagentX/imessage-exporter/blob/develop/docs/features.md).

**Reply by veitch123** [reply]
Awesome. I've got a 400gb messages library. Reckon that'll be a problem? I wonder whether i can also get it to parse the Chatology backups I had before iCloud messages was a thing.

**Reply by ReagentX** [reply]
It should be fine if you have enough disk space to copy it all. If you run into any issues, please open an issue!

I don't support whatever Chatology is. This project addresses iMessage data only.

**Reply by veitch123** [reply]
Sure thing. 
Chatology was a way of archiving and searching imessages before it was possible or easy. I forget when exactly it came in but it was discontinued in 2020. I still have my archives though. 

![screenshot-1](https://user-images.githubusercontent.com/24921035/219774751-519210a9-a83c-4d8c-9957-b56729e9b090.jpeg)

https://9to5mac.com/2020/09/09/chatology-for-mac-discontinued/

**Reply by ReagentX** [reply]
> Chatology was a way of archiving and searching imessages before it was possible or easy

I would argue that is still not easy! 

This software looks cool. I am sad that it was discontinued.
